projects/novo-examples/src/components/non-ideal-state/non-ideal-state-loading-usage/non-ideal-state-loading-usage-example.html

```diff
@@ -0,0 +1,6 @@
+<novo-non-ideal-state>
+  <novo-loading></novo-loading>
+  <novo-text lineLength="large" marginAfter>We are currently experiencing technical difficulites and your wait time is
+    taking a bit longer than expected. Please be patient. </novo-text>
```

Comment:
Super minor - I know this is just an example, but you can strike the second sentence. It's totally fine with just the first. The general guidance from UX we've received in the past is to never say Please to the user. Also, this makes it sound like they are impatient.